lib/datapackage.py

```diff
@@ -139,25 +155,24 @@ def create_json():
                             raise
                         col_obj["size"] = col_list[2]
                     table["schema"]["fields"].append(col_obj)
-
                 except:
                     print("Exception occured. Check the input format again.\n")
                     pass
 
-                col = input()
+                col = input().strip()
 
-            isCT = input("Add crosstab columns? (y,N): ")
-            if isCT.strip().lower() in ["y", "yes"]:
-                ct_column = input("Crosstab column name: ")
-                while ct_column.strip() == "":
+            isCT = input("Add crosstab columns? (y,N): ").strip()
+            if isCT.lower() in ["y", "yes"]:
+                ct_column = input("Crosstab column name: ").strip()
+                while ct_column == "":
                     print("Empty column name. Try again.\n")
-                    ct_column = input("Crosstab column name: ")
+                    ct_column = input("Crosstab column name: ").strip()
                 ct_names = []
-                print("Enter names of crosstab column values (Press return after each name)")
-                name = input()
-                while name.strip() != "":
+                print("Enter names of crosstab column values (Press return after each name):\n")
+                name = input().strip()
+                while name != "":
                     ct_names.append(name)
-                    name = input()
+                    name = input().strip()
 
                 table['schema']['ct_column'] = ct_column
                 table['schema']['ct_names'] = ct_names
```

Comment:
line [182](https://github.com/weecology/retriever/pull/599/files#diff-fe19ce9787a421223628d4355dc2f5e4R182) 

```
     file_name = contents['name'] + ".json"
     with open(os.path.join(HOME_DIR, 'scripts', file_name), 'w') as output_file:
         json.dump(contents, output_file, sort_keys=True, indent=4,
                   separators=(',', ': '))
         output_file.write('\n')
         print("\nScript written to " + file_name)
         output_file.close()
```

line [352](https://github.com/weecology/retriever/pull/599/files#diff-fe19ce9787a421223628d4355dc2f5e4R352)

```
     file_name = contents['name'] + ".json"
     with open(os.path.join(HOME_DIR, 'scripts', file_name), 'w') as output_file:
         json.dump(contents, output_file, sort_keys=True, indent=4,
                   separators=(',', ': '))
         output_file.write('\n')
         print("\nScript written to " + os.path.join(HOME_DIR, 'scripts', file_name))
         output_file.close()
```

These can be refactored